packages/smart-wallet/src/smartWallet.js

```diff
@@ -673,7 +673,7 @@ export const prepareSmartWallet = (baggage, shared) => {
           const { invitationPurse, address } = state;
 
           const brandToPurses = getBrandToPurses(walletPurses, self);
-          trace(`Found ${brandToPurses.values()} purse(s) for ${address}`);
+          trace(`Found purse(s) for ${address}`);
```

Comment:
Is this change to debug logging in the smart wallet meant to go into a "testing" PR?

It's also a change I don't see on master.

Reply:
please revert